shared-bindings/busio/I2C.c

```diff
@@ -67,10 +67,11 @@
 STATIC mp_obj_t busio_i2c_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
     busio_i2c_obj_t *self = m_new_obj(busio_i2c_obj_t);
     self->base.type = &busio_i2c_type;
-    enum { ARG_scl, ARG_sda, ARG_frequency, ARG_timeout };
+    enum { ARG_scl, ARG_sda, ARG_setpullup, ARG_frequency, ARG_timeout };
     static const mp_arg_t allowed_args[] = {
         { MP_QSTR_scl, MP_ARG_REQUIRED | MP_ARG_OBJ },
         { MP_QSTR_sda, MP_ARG_REQUIRED | MP_ARG_OBJ },
+        { MP_QSTR_int, MP_ARG_KW_ONLY | MP_ARG_INT, {.u_int = 0}  },
```

Comment:
```suggestion
        { MP_QSTR_pullup, MP_ARG_KW_ONLY | MP_ARG_BOOL, {.u_bool = false}  },
```
Let's use `boolean` instead of `int` for this.